stdlib/3/itertools.pyi

```diff
@@ -42,7 +42,7 @@ def groupby(iterable: Iterable[_T],
 @overload
 def islice(iterable: Iterable[_T], stop: int) -> Iterator[_T]: ...
```

Comment:
This argument should also be Optional:
```
In [137]: islice([1, 2], None)
Out[137]: <itertools.islice at 0x11d435cc8>

In [138]: list(_)
Out[138]: [1, 2]
```

Reply:
Good catch. I've updated `stdlib/3/itertools.pyi`. Should I include the same change to `stdlib/2/itertools.pyi` in this PR?

Reply:
Yes, please. The argument name should be `stop` not `start` though, since `islice(it, 2)` is equivalent to `islice(it, 0, 2)`.

Also, all arguments to islice are positional-only: `islice(it, start=2)` throws an error. We can express this in the stub by prefixing the argument name with `__` (https://www.python.org/dev/peps/pep-0484/#positional-only-arguments).